src/NAME/DependenciesReader.cs

```diff
@@ -248,6 +248,13 @@ private static IConnectionStringProvider ParseConnectionStringProvider(JsonNode
                         if (string.IsNullOrEmpty(file))
                             throw new ArgumentNullException("file", "The file must be specified.");
                         provider = new XpathConnectionStringProvider(pathMapper.MapPath(file), key);
+                        break;
+                    }
+                case SupportedConnectionStringLocators.EnvironmentVariable:
+                    {
+                        key = node["expression"]?.Value;
```

Comment:
I think we should extract the environment variable name from the `"key"` element instead of `"expression"`, just like what we did for the AppSettings provider. I think `"expression"` only makes sense for JSONPath and XPath.